Makefile

```diff
@@ -3,15 +3,37 @@
 SUBDIR=	linuxkpi	\
 	lindebugfs	\
 	drm		\
-	vmwgfx		\
-	i915		\
+	${_vmwgfx}	\
+	${_i915}	\
 	amd		\
 	radeon		\
 	${_staging}	
 
+.if ${MACHINE_CPUARCH} == "amd64" || ${MACHINE_CPUARCH} == "i386"
+_i915 =		i915 
+_vmwgfx =	vmwgfx
+.endif
 
 .if defined(STAGING)
 _staging=	staging
 .endif
 
+
```

Comment:
I'm not sure I understand the intentions of this, and if this is something we want in the port.

I believe that installing something from ports that fiddles with base source is a big no-no, but perhaps I'm misunderstanding something.